.circleci/config.yml

```diff
@@ -0,0 +1,90 @@
+version: 2.1
```

Comment:
q: The pipeline is executed here: https://app.circleci.com/pipelines/github/localstack-samples/sample-serverless-image-resizer-s3-lambda?branch=circleci-project-setup
Looking for the pipeline brought me to the question: Would it make sense to add links or badges to the top of the readme linking to the different CI environments this repo integrates with?

Reply:
Thought about it, it's an option we can add either now or later.

Reply:
Either way works fine for me :)